pkg/bpf/bpf_linux.go

```diff
@@ -0,0 +1,413 @@
+// Copyright 2016-2018 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build linux
```

Comment:
Nit: `// +build linux` is redundant here (and in other files bellow), as the build constraint is derived from the file name.

Reply:
Yes, it's done to be consistent with the rest of the code